lib/components/viewers/stop-viewer.js

```diff
@@ -481,7 +481,9 @@ const mapStateToProps = (state) => {
   const { autoRefreshStopTimes = true, favoriteStops } = state.user.localUser
   const stopLookup = transitIndex.stops
   const stopData = stopLookup[ui.viewedStop.stopId]
-  const nearbyStops = stopData?.nearbyStops?.map((stopId) => stopLookup[stopId])
+  const nearbyStops = Array.from(new Set(stopData?.nearbyStops))?.map(
```

Comment:
Maybe it could be a bit cleaner if we broke the set onto a different line? 
like: `uniqueNearbyStops = Array.from(new Set(stopData?.nearbyStops))` 
but I'm not too sure, so up to you